configure.ac

```diff
@@ -72,6 +72,20 @@ AC_TYPE_UINT8_T
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([strdup])
 
+LIBVIRT_REQUIRED="1.2.2"
+
+AC_ARG_WITH([libvirt],
+            [AS_HELP_STRING([--without-libvirt],
+                            [run hyperd with libvirt])],
+            [with_libvirt=no],[with_libvirt=yes])
+
+if test "$with_libvirt" = yes; then
+	PKG_CHECK_MODULES([libvirt], [libvirt >= $LIBVIRT_REQUIRED], [], [
+		AC_MSG_ERROR([libvirt-devel >= $LIBVIRT_REQUIRED is required or reconfigure runv with --without-libvirt])])
```

Comment:
it won't fail, if there is no libvirt-dev or the version is incorrect, you should reconfigure hyper/runv with --without-libvirt, please check https://github.com/hyperhq/runv/pull/96.

We want xen&libvirt driver being available by default, so we outthrow warning if there is no xen or libvirt available,and user can use --without- option to reconfigure hyper/runv.